frontend/apps/marketing/src/components/actionBlocks/defaultActionBlock/ActionBlock.tsx

```diff
@@ -16,6 +18,7 @@ export type ActionBlockContentfulProps = ActionBlockProps & {
   primaryButton: LinkEntry;
   secondaryButton: LinkEntry;
   background: EntryFields.Text;
+  publishedDate: EntryFields.Date | undefined;
```

Comment:
nit: 
Isn't it possible to simplify this to this? :

```suggestion
  publishedDate?: EntryFields.Date;
```